Kajabi/sage-lib
# [DSS-99] Update icon font

## Description

Adds the following icons:

- `list-details` (exists in current font but has not been mapped)
- `logo-facebook`
- `logo-instagram`
- `logo-linkedin`
- `logo-tiktok`
- `logo-twitter`

TODO:
- [x] upload to CDN
- [x] revise icon path

## Screenshots

|  name  |  icon  |
|--------|--------|
|`list-details`|![list-details](https://user-images.githubusercontent.com/816579/190692116-62289c3f-42f7-4860-988f-77d8beb51c07.png)|
|`logo-facebook`|![logo-facebook](https://user-images.githubusercontent.com/816579/190692151-2e9b1f58-be62-4f95-a492-5d2e6b6fe2e6.png)|
|`logo-instagram`|![logo-instagram](https://user-images.githubusercontent.com/816579/190692175-374a495e-841f-42c4-a3fa-f02c734e8b2c.png)|
|`logo-linkedin`|![logo-linkedin](https://user-images.githubusercontent.com/816579/190692214-ecce3d4c-80f4-43cf-abba-9ca2fbc7ccc7.png)|
|`logo-tiktok`|![logo-tiktok](https://user-images.githubusercontent.com/816579/190692244-41ea20f8-bf25-4f79-baec-b615e999099c.png)|
|`logo-twitter`|![logo-twitter](https://user-images.githubusercontent.com/816579/190692291-8bb3f7c1-eaca-4fef-869f-f66361287b3c.png)|


## Testing in `sage-lib`
1. Navigate to http://localhost:4000/pages/foundations/icon
2. Verify all icons are rendering as expected


## Testing in `kajabi-products`
1. (LOW) Added new icons. No impact expected.


## Related
- [DSS-99](https://kajabi.atlassian.net/browse/DSS-99)
